<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingresar Producto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_09">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stock Máximo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxCategoriaProductos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_07">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*Stock :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarProveedor">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/add-1.png</normaloff>../icon/add-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>13</width>
      <height>13</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDescripcion">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>501</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxProveedores">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_04">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*Precio Costo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_06">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Proveedor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_03">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Categoría:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_08">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stock Crítico:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_01">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*Código Artículo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregaCategoria">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/add-1.png</normaloff>../icon/add-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>13</width>
      <height>13</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_02">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*Descripción     :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_05">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*Precio Venta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>INGRESAR UN NUEVO ARTICULO A SISTEMA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/save.png</normaloff>../icon/save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/error.png</normaloff>../icon/error.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtStockCritico">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtStockMaximo">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtStock">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtPrecioVenta">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtPrecioCosto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtCodigo</tabstop>
  <tabstop>txtDescripcion</tabstop>
  <tabstop>txtPrecioCosto</tabstop>
  <tabstop>txtPrecioVenta</tabstop>
  <tabstop>txtStock</tabstop>
  <tabstop>txtStockCritico</tabstop>
  <tabstop>txtStockMaximo</tabstop>
  <tabstop>cbxCategoriaProductos</tabstop>
  <tabstop>cbxProveedores</tabstop>
  <tabstop>btnGuardar</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>btnAgregaCategoria</tabstop>
  <tabstop>btnAgregarProveedor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
